runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningCoalBagOverlay.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 Jarred Vardy <jarredvardy@gmail.com>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.mining;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import javax.inject.Inject;
+import net.runelite.api.ItemID;
+import net.runelite.api.Point;
+import net.runelite.api.widgets.WidgetItem;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.overlay.WidgetItemOverlay;
+
+public class MiningCoalBagOverlay extends WidgetItemOverlay
+{
+	private final MiningPlugin plugin;
+	private final MiningConfig config;
+
+	@Inject
+	MiningCoalBagOverlay(MiningPlugin plugin, MiningConfig config)
+	{
+		showOnInventory();
+		showOnInterfaces();
+		this.plugin = plugin;
+		this.config = config;
+	}
+
+	@Override
+	public void renderItemOverlay(Graphics2D graphics, int itemId, WidgetItem itemWidget)
+	{
+		if (!config.showCoalBagOverlay())
+		{
+			return;
+		}
+
+		if (itemId != ItemID.COAL_BAG && itemId != ItemID.COAL_BAG_12019)
+		{
+			return;
+		}
+
+		graphics.setFont(FontManager.getRunescapeSmallFont());
+		graphics.setColor(Color.YELLOW);

```

Comment:
```suggestion
		graphics.setColor(Color.WHITE);
```
To keep it consistent with how Item Charges are displayed